apps/website/.vuepress/theme/global-components/ClrColorListSet.vue

```diff
@@ -1,24 +1,24 @@
 <template>
   <div>
-    <div class="clr-form-control">
-      <label class="clr-control-label">Preview colors in: </label>
-      <div class="clr-control-container clr-control-inline">
-        <div class="clr-radio-wrapper">
-          <input type="radio" :id="'hsl-radio-' + listSetId" value="hsl" class="clr-radio" v-model="picked" />
-          <label :for="'hsl-radio-' + listSetId" class="clr-control-label">HSL</label>
-        </div>
-        <div class="clr-radio-wrapper">
+    <cds-form-group layout="horizontal-inline" cds-layout="m-t:md">
+      <cds-radio-group layout="horizontal-inline">
+        <label>Preview colors in:</label>
+        <cds-radio>
+          <label :for="'hsl-radio-' + listSetId">HSL</label>
+          <input :id="'hsl-radio-' + listSetId" type="radio" value="hsl" v-model="picked" />
+        </cds-radio>
+
+        <cds-radio>
+          <label :for="'hex-radio-' + listSetId">HEX</label>
           <input type="radio" :id="'hex-radio-' + listSetId" value="hex" class="clr-radio" v-model="picked" />
```

Comment:
Not sure if the ids are used to for other functionality on the page but the cds form controls will auto generate and assign a id for the input and label.